src/modules/profile/sagas.ts

```diff
@@ -43,5 +57,62 @@ export function createProfileSaga({ peerUrl }: CreateProfileSagaOptions) {
     yield put(loadProfileRequest(action.payload.wallet.address))
   }
 
+  /**
+   * Handles the action to set the description of a user's profile.
+   * This handler gets the first profile entity of a given address
+   * and then rebuilds it with the specified description to deploy it
+   * again.
+   *
+   * @param action - The action that triggered the handler.
+   */
+  function* handleSetProfileDescription(
+    action: SetProfileAvatarDescriptionRequestAction
+  ) {
+    try {
+      const { address, description } = action.payload
+
+      const entity: ProfileEntity = yield call(
+        [entities, 'getProfileEntity'],
```

Comment:
Just out of curiosity, why using this notation instead of just `entities.getProfileEntity`? It seems like a more complicated way of doing the same, i dunno what am i missing.

Reply:
This is [the proper way](https://redux-saga.js.org/docs/api/#callcontext-fn-args) of using the call method with a context. As we're calling the `getProfileEntity` method from the `entities` instance, we need to set the binding to the instance of that class for the call to work. I've noted that we used `yield call(() => x())` in some places, but using it that way is not how it's meant to be used, as it makes it hard to test because the call object being built has a reference to the lambda function instead of the method that we want to check for.

Reply:
Gotcha, thanks for the explanation :)